### ISSUE TITLE
nlbits and nmbits reset requirements

### ISSUE THREAD

**Issue by billhuffman** [riscv/riscv-fast-interrupt#177]
Since we now require nlbits to be either 0 or 8, do we still want to require it to be reset to 0?  I'm thinking it doesn't need a reset requirement - and all the more so because many will want to support only 0 or only 8.   

Along the same lines, do we need to make a reset requirement on nmbits either?

**Comment by Kevin-Andes** [comment]
Yes, your comments make sense. The reset value should be implementation dependent and may even be hardwired to some fixed value...
